### PR TITLE
Fix/back button and explicit breadcrumb type links

### PR DESCRIPTION
**Pull request by dorianjp**
# Summary | Résumé

Trello
https://trello.com/c/98nyxaik/798-back-button-goes-to-the-last-page-from-date-time-selection-pages

* changed some back buttons to breadcrumb style explicit links
* Added js backbutton support
* Fixed alert wizard to no longer need session primer for location ID, initial page for wizard includes location id in the url

# Test instructions | Instructions pour tester la modification

* QR Alert datetime page should not show back, instead explicit link back to details of place/event
* Profile page also has explicit link back to new search
* alert details has regular back link using js

### CODE REVIEW

File: outbreaks/templates/details.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 {% block content %}
     <div class="content-back-link">
-        <a href="{% url 'outbreaks:history' %}" class="chevron-left">{% trans "Back" %}</a>
+        <a href="#" class="chevron-left back-inline-link">{% trans "Back" %}</a>
```
Comment (dorianjp) [inline outbreaks/templates/details.html:11]: we don't always get here from 'history', now fixed

File: outbreaks/templates/profile.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 {% block content %}
     <div class="content-back-link">
-        <a href="{% url 'outbreaks:search' %}" class="chevron-left">{% trans "Back" %}</a>
+        <a href="{% url 'outbreaks:search' %}" class="chevron-left">{% trans "Find QR places and events" %}</a>
```
Comment (dorianjp) [inline outbreaks/templates/profile.html:10]: more explicit link

File: outbreaks/urls.py [hunk 3]
```diff
@@ -20,12 +20,12 @@
         name="location_history",
     ),
     path(
-        "datetime",
+        "<uuid:pk>/datetime",
```
Comment (dorianjp) [inline outbreaks/urls.py:23]: fix to get rid of session primer on wizard

File: outbreaks/views.py [hunk 4]
```diff
@@ -156,23 +156,18 @@ def get(self, request, *args, **kwargs):
...
-            ).count()
+        # Fetch the location
+        location = Location.objects.get(id=self.kwargs["pk"])
+        context["location"] = location
+        context["map_link"] = "https://maps.google.com/?q=" + str(location)
+        location.category = dict(location_choices)[location.category]
+        # Cache the location for the next steps
+        context["alert_location"] = self.kwargs["pk"]
 
-        except Location.DoesNotExist:
```
Comment (dorianjp) [inline outbreaks/views.py:173]: can't get to this page without a location, so if we 500 here now, we would want to know

File: outbreaks/views.py [hunk 5]
```diff
@@ -182,9 +177,11 @@ class DatetimeView(PermissionRequiredMixin, Is2FAMixin, View):
     template_name = "datetime.html"
 
     def get(self, request, *args, **kwargs):
-        # Ensure we have a cached location
-        if "alert_location" not in request.session:
-            return redirect(reverse_lazy("outbreaks:search"))
+        location_id = str(kwargs.get('pk'))
```
Comment (dorianjp) [inline outbreaks/views.py:180]: new way to clear saved selected dates if skipping out of the wizard mid-way when picking a new location

File: profiles/static/js/script.js [hunk 6]
```diff
@@ -38,3 +38,12 @@ if (validationSummary) {
 }
 
 document.cookie = 'browserTimezone=' + Intl.DateTimeFormat().resolvedOptions().timeZone + '; path=/';
+
+function navBack() {
```
Comment (dorianjp) [inline profiles/static/js/script.js:42]: fixes csp issues

**Comment by dorianjp** [comment]
[Back button goes to the last page (from date & time selection pages)](https://trello.com/c/98nyxaik/798-back-button-goes-to-the-last-page-from-date-time-selection-pages)

**Review by dsamojlenko — APPROVED** [review]
LGTM

**Pull request merged**